src/pages/index.js

```diff
@@ -101,10 +101,6 @@ profileImageButton.addEventListener("click", () => {
   profileImageModal.open();
 });
 
-profileImageSaveButton.addEventListener("click", () => {
-  handleProfileImageForm();
-});
-
 function processLikeClick(card) {
```

Comment:
**Might be better**

To have access to any method of the class in `index.js` you can pass `this` to the call:

```js

        this._likeButton.addEventListener('click', () => {
            this._handleLike(this);
        });


```

And now in `index.js` you can call the methods like this:


```js


function addLike(cardInstance) {
        api.likeCard(cardInstance.getId())
            .then((res) => {
               cardInstance.addLike(res)
            })
            .catch((err) => {
                console.error(err);
            });
}


```

The same thing is with deleting cards